webpack.config.js

```diff
@@ -77,7 +77,20 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
           : filename;
       },
       writeToDisk: true,
+      integrity: isProductionEnv,
       output: 'manifest.json',
+      transform(manifest) {
+        const srcIntegrity = {};
```

Comment:
This changes the output from something like...

```json
{
  "whatever.js": {
    "src": "/packs/whatever-u09n1lo.js",
    "integrity": "sha256-..."
   }
}
```

...to a more consumable...

```json
{
  "integrity": {
    "/packs/whatever-u09n1lo.js": "sha256-..."
   }
}
```